### PR TITLE
update-work

### LINKED ISSUE #539: Add missing clients to Work
## Add Pages

- [x] Sass - add playground and website
- [x] GSM - add
- [x] Google - Flesh out info on the Google Chrome project @mirisuzanne 
- [x] Sass - Review Sass project content that James contributed @SondraE    

## Checklist

- [ ] Get quote and pic from Ryan @SondraE _(followed up April 15)_
- [x] Get quote (about design systems & Google) and pic from Nicole @mirisuzanne
- [x] Add logos to front page @SondraE 
- [x] Get quote and pic from Natalie @mirisuzanne 
- [x] Blurb about what we did and tools/languages we used for Sass project @jamesnw

### PR DESCRIPTION
**Pull request by SondraE**
## Description
Adding missing clients to the Work list

## Related Issue(s)

- Fixes #539 

## Steps to test/reproduce
- /work/
- Sass Playground & Website
- Estimating Engine

### CODE REVIEW

File: content/work/sass.md
Comment (jgerigmeyer) [inline content/work/sass.md:1]: If this is Sass Website only, probably update the filename to `sass-website.md` or something?

**Review by jgerigmeyer — COMMENTED** [review]
Can we feature some of these new clients on the home page?

File: content/tools/blend.md
Comment (mirisuzanne) [reply content/tools/blend.md:36]: I think we link to `/sass/` unless we're talking about something we built for them as a client? I don't know…

Comment (mirisuzanne) [reply content/tools/blend.md:36]: oh nooooooo. I don't have a good answer. Should we try and figure out a way to merge the two?

Comment (jgerigmeyer) [reply content/tools/blend.md:36]: I'd be curious what @SondraE thinks. My initial leaning might be to leave both in place, but prefer linking to `/work/sass-core/` -- and could even mention on that page that you've been a core contributor since 2016.

**Comment by SondraE** [comment]
@mirisuzanne 

> Can we feature some of these new clients on the home page?

I think we should either add another set of four logos to the featured clients section (Chrome, Google, Sass, GMS?) or if that doesn't quite make sense, then I think SRAM and Phamaly would be good candidates to replace. SRAM because it was a long time ago and Phamaly because we aren't looking for more WP work.

File: content/tools/blend.md
Comment (SondraE) [reply content/tools/blend.md:36]: @mirisuzanne I share Jonny's leaning. And I do think that page should indicate that you've been a core contributor.

File: content/blog/index.njk
Comment (jgerigmeyer) [inline content/blog/index.njk:9]: @mirisuzanne Because we use a custom sort (by `end` date), this needs to be sorted before pagination -- without this, the TimeDesigner case study was paginated with 2017 posts, but displayed as a 2022 post.

File: content/work/sass-site.md
Comment (jgerigmeyer) [inline content/work/sass-site.md:85]: Do we want to mention Eleventy here too?

**Comment by SondraE** [comment]
@mirisuzanne Well...I tried to add Google's logo to the homepage. On index.njk, I replaced `/work/phamaly/` with   `/work/google/`. We already have an svg of google's logo. On the UI Fund page I added `logo: google`. But apparently there's something else I need to change, and I'm not sure what it is.

**Comment by jgerigmeyer** [comment]
@SondraE `/work/google/` doesn't exist, which is why it broke. I replaced it with `work/ui-fund/`, which is where I assume you wanted to link to?

I also re-ordered the logos a bit, but feel free to replace that if you like.

**Comment by mirisuzanne** [comment]
@SondraE do we need to wait for quotes, or is this ready for final review?

**Comment by SondraE** [comment]
@mirisuzanne I don't think we have a way to publish work without a quote at this point, right? I followed up with Ryan this morning, and he said he doesn't have a quote for us yet, but he's planning to get something together in the next few days. I think we're waiting for that.

**Comment by mirisuzanne** [comment]
@SondraE fine to wait if we want - but we did change it so we only need a quote OR an image to publish it on the work page.

File: content/work/ui-fund.md
Comment (jgerigmeyer) [inline content/work/ui-fund.md]: Do we want to add URLs for either of these people?

Comment (mirisuzanne) [reply content/work/ui-fund.md]: We could. Links to the tweets, or links to the projects?

Comment (jgerigmeyer) [reply content/work/ui-fund.md]: Either way -- what have we done elsewhere? It looks to me like we're mostly linking to projects/linkedin.

**Comment by mirisuzanne** [comment]
@jgerigmeyer I added urls for those quotes

@SondraE I added Nicole's quotes to a few of the pages. I also added a speaking quote to the home page. See what you think.

File: content/work/ui-fund.md
Comment (jgerigmeyer) [inline content/work/ui-fund.md:72]: Now that we added sponsors to this page, should we just link internally?

```suggestion
  We were [sponsored](#open-source-sponsors)
```

File: content/work/sass-core.md
Comment (jgerigmeyer) [inline content/work/sass-core.md:29]: Do we want to show this quote on this page?

**Review by jgerigmeyer — APPROVED** [review]

File: content/work/sass-core.md
Comment (jgerigmeyer) [reply content/work/sass-core.md:29]: Added in https://github.com/oddbird/oddleventy/pull/540/commits/259a7e0ec3a14b8a844a8bcf441e65929334c96e

**Comment by SondraE** [comment]
@mirisuzanne 
This is all looking really good to me. 

The only thing that looks a little strange to me is the way the Services and Language & Tools lists look when there is no image helping to orient them in space. 
Screenshot 2024-04-16 at 1 36 28 PM


Below are the two layouts for those columns on the other projects:
Screenshot 2024-04-16 at 1 34 12 PM

Screenshot 2024-04-16 at 1 38 45 PM

Could we either add images to the new projects just above those lists so that the pattern is consistent across projects or, if there are no relevant images to add, then could we just stack those lists instead of using the two column layout?

**Comment by mirisuzanne** [comment]
@SondraE How's that look? (I also added a Nicole quote to the planning services page)

**Comment by SondraE** [comment]
@mirisuzanne Ship it!

**Pull request merged**
